js/script.js

```diff
@@ -1,223 +1,227 @@
+    window.setTimeout(() => {
+        document.documentElement.classList.remove('transition')
+    }, 300)
+}
+
+// menu marker animation
+
+
+function menuChange() {
+    if ($('#hori').is(':visible')) {
+        var marker = document.querySelector('#marker');
+        var item = document.querySelectorAll('nav a');
+
+        function indicator(e) {
+            marker.style.left = e.offsetLeft + "px";
+            marker.style.width = e.offsetWidth + "px";
+        }
+
+        item.forEach(link => {
+            link.addEventListener('mouseover', (e) => {
+                indicator(e.target);
+            })
+        })
+    }
+}
+
+if ($('#marker').length) {
+    $(window).resize(function () {
+        marker.style.left = 0;
+        marker.style.width = 0;
+        menuChange();
+    });
+};
+menuChange();
+
+
+function getCurrentFileName() {
+    var pagePathName= window.location.pathname;
+    return pagePathName.substring(pagePathName.lastIndexOf("/") + 1);
+}
+
+// Update CSS upon clicking new cursors
+var root = "../"
+if (getCurrentFileName().includes("index.html")) {
+    root = ""
+}
+
+$(document).ready(function () {
+    $('#grogu').on('click', function () {
+        $('body').css('cursor', `url(${root}assets/cursors/grogu_cursor.cur), auto`);
+    })
+})
+
+$(document).ready(function () {
+    $('#cashew').on('click', function () {
+        $('body').css('cursor', `url(${root}assets/cursors/cashew_cursor.cur), auto`);
+    })
+})
+
+$(document).ready(function () {
+    $('#bigchung').on('click', function () {
+        $('body').css('cursor', `url(${root}assets/cursors/big_chungus_cursor.cur), auto`);
+    })
+})
+
+$(document).ready(function () {
+    $('#ugchung').on('click', function () {
+        $('body').css('cursor', `url(${root}assets/cursors/ugandan-chungus-cursor.cur), auto`);
+    })
+})
+
+$(document).ready(function () {
+    $('#amus').on('click', function () {
+        $('body').css('cursor', `url(${root}assets/cursors/among-us-pointer.cur), auto`);
+    })
+})
+$(document).ready(function () {
+    $('#cureset').on('click', function () {
+        $('body').css({'cursor': ''});
+    })
+
+    $('a[href*="https://elfsight.com/event-calendar-widget/?utm_source=websites&utm_medium=clients&utm_content=event-calendar&utm_term=www.pasa.website&utm_campaign=free-widget"]').parent('div').remove();
+})
+
+// Copy email and link other social media buttons
+$("#email").click(function(){
+
+    copyToClipboard("physastr@ucalgary.ca");
+    $("#emailid").html("Email Copied to Clipboard");
+
+})
+
+$("#instagram").click(function(){
+
+    window.open("https://www.instagram.com/ucalgarypasa");
+
+})
+
+$("#facebook").click(function(){
+
+    window.open("https://www.fb.com/UCalgaryPASA");
+
+})
+
+function copyToClipboard(text) {
+    var $temp = $("<input>");
+    $("body").append($temp);
+    $temp.val(text).select();
+    document.execCommand("copy");
+    $temp.remove();
+}
+
+console.log()
```

Comment:
Why do we have a random console.log here?

Reply:
I noticed a bunch of depreciated code, old comments, and useless sections too that I think we can probably address in a future PR.

Reply:
I'll remove this one for the time being though.